tet.rs/src/tetris/board/piece/piece_matrix.rs

```diff
@@ -0,0 +1,136 @@
+pub type MatrixT = u16;
+
+use crate::tetris::board::piece::{
+    piece_position::PositionT, piece_rotation::PieceRotation, piece_type::PieceType,
+};
+
+const IN: u16 = 0b0000111100000000;
+const IE: u16 = 0b0010001000100010;
+const IS: u16 = 0b0000000011110000;
+const IW: u16 = 0b0100010001000100;
+
+const ON: u16 = 0b110110000;
+const OE: u16 = 0b011011000;
+const OS: u16 = 0b000011011;
+const OW: u16 = 0b000110110;
+
+const JN: u16 = 0b100111000;
+const JE: u16 = 0b011010010;
+const JS: u16 = 0b000111001;
+const JW: u16 = 0b010010110;
+
+const LN: u16 = 0b001111000;
+const LE: u16 = 0b010010011;
+const LS: u16 = 0b000111100;
+const LW: u16 = 0b110010010;
+
+const SN: u16 = 0b011110000;
+const SE: u16 = 0b010011001;
+const SS: u16 = 0b000011110;
+const SW: u16 = 0b100110010;
+
+const ZN: u16 = 0b110011000;
+const ZE: u16 = 0b001011010;
+const ZS: u16 = 0b000110011;
+const ZW: u16 = 0b010110100;
+
+const TN: u16 = 0b010111000;
+const TE: u16 = 0b010011010;
+const TS: u16 = 0b000111010;
+const TW: u16 = 0b010110010;
+
+pub struct PieceMatrix {}
```

Comment:
We rarely use empty structs as a way to create namespaces, creating a submodule or putting everything in its own file is preferred.

Reply:
Perfect ! That'll make things easier !